app/webpacker/components/Panel/pages/AnonymizationScriptPage/index.jsx

```diff
@@ -1,24 +1,55 @@
 import React from 'react';
-import WcaSearch from '../../../SearchWidget/WcaSearch';
+import { FormField, FormGroup, Radio } from 'semantic-ui-react';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
 import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
 import useInputState from '../../../../lib/hooks/useInputState';
-import AnonymizationTicketWorkbench from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbench';
+import AnonymizationTicketWorkbenchForWrt from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt';
+
+const AVAILABLE_MODELS = [
+  {
+    id: SEARCH_MODELS.user,
+    name: 'User',
+  },
+  {
+    id: SEARCH_MODELS.person,
+    name: 'Person',
+  },
+];
 
 export default function AnonymizationScriptPage() {
-  const [user, setUser] = useInputState();
+  const [activeModelIndex, setActiveModelIndex] = useInputState(0);
```

Comment:
You can use this for a `const activeModel = useMemo(() => AVAILABLE_MODELS[activeModelIndex], [activeModelIndex]);`

Reply:
I don't think `useMemo` is necessary. The computation is not expensive, and the model object is defined outside the component so will be the same object reference each render (unless the index changes).

Reply:
Oops ya, you are right. Removed it.